src/modules/dashboard/components/Dialogs/ManageGnosisSafeDialog/ManageGnosisSafeDialog.tsx

```diff
@@ -41,9 +41,13 @@ const MSG = defineMessages({
   },
   permissionText: {
     id: 'dashboard.ManageGnosisSafeDialog.permissionsText',
-    defaultMessage: `You must have the {permission} permissions in the
+    defaultMessage: `You must have the {permissionsList} permissions in the
       relevant teams, in order to take this action`,
   },
+  adminFundingPermissions: {
+    id: 'dashboard.AdvancedDialog.gnosisPermissionsList',
```

Comment:
Should this be dashboard.ManageGnosisSafeDialog.adminFundingPermissions since its in the ManageGnosisSafeDialog component and the label is  adminFundingPermissions?

Reply:
Correct, I've updated it